### DISCUSSION TITLE
关于mpv的中文输入法乱入

### DISCUSSION THREAD

**Discussion by mei-iem** [hooke007/mpv_PlayKit#451] [mpv 通用]
找了一圈，好像没人说mpvlazy的中文输入法
简单来说就是mpvlazy不会禁用中文输入法，这导致想要使用应用内快捷键时需要切换为英文输入法
（顺带一提，我记得net好像是没有这个问题
带入一下，其实还挺难受的，所以来问问各位有没有能解决这个问题的脚本
（或者说大佬有意向解决这个小问题

**Comment by hooke007** [comment]
这说过了。 
这也不是lazy的问题
https://github.com/hooke007/MPV_lazy/discussions/442#discussioncomment-9108826
https://github.com/hooke007/MPV_lazy/discussions/399#discussion-6021864
https://github.com/mpv-player/mpv/issues/13666
